apps/emqx/src/emqx_config.erl

```diff
@@ -954,3 +956,43 @@ put_config_post_change_actions(_Key, _NewValue) ->
 
 config_files() ->
     application:get_env(emqx, config_files, []).
+
+remove_computed_fields(Config) ->
```

Comment:
maybe this should be a flag for the `Opts` arg for `hocon_tconf:map_translate/3`, so it will will not try to return computed field?.

Reply:
it would have to be for `hocon_tconf:check_plain` in this case, but could be an alternative.

Reply:
https://github.com/emqx/hocon/pull/306